static/app/views/issueList/overview.tsx

```diff
@@ -1023,87 +1023,92 @@ class IssueListOverview extends React.Component<Props, State> {
-                queryCount={modifiedQueryCount}
-                displayCount={displayCount}
-                onSelectStatsPeriod={this.onSelectStatsPeriod}
-                onMarkReviewed={this.onMarkReviewed}
-                onDelete={this.onDelete}
-                statsPeriod={this.getGroupStatsPeriod()}
-                groupIds={groupIds}
-                allResultsVisible={this.allResultsVisible()}
-                displayReprocessingActions={displayReprocessingActions}
+                savedSearch={savedSearch}
+                sort={this.getSort()}
+                display={this.getDisplay()}
+                onDisplayChange={this.onDisplayChange}
+                onSortChange={this.onSortChange}
+                onSearch={this.onSearch}
+                onSidebarToggle={this.onSidebarToggle}
+                isSearchDisabled={isSidebarVisible}
+                tagValueLoader={this.tagValueLoader}
+                tags={tags}
+                selectedProjects={selection.projects}
               />
-              <PanelBody>
-                <ProcessingIssueList
+
+              <Panel>
+                <IssueListActions
                   organization={organization}
-                  projectIds={projectIds}
-                  showProject
+                  selection={selection}
+                  query={query}
+                  queryCount={modifiedQueryCount}
+                  displayCount={displayCount}
+                  onSelectStatsPeriod={this.onSelectStatsPeriod}
+                  onMarkReviewed={this.onMarkReviewed}
+                  onDelete={this.onDelete}
+                  statsPeriod={this.getGroupStatsPeriod()}
+                  groupIds={groupIds}
+                  allResultsVisible={this.allResultsVisible()}
+                  displayReprocessingActions={displayReprocessingActions}
                 />
-                {this.renderStreamBody()}
-              </PanelBody>
-            </Panel>
-            <StyledPagination
-              caption={tct('Showing [displayCount] issues', {
-                displayCount,
-              })}
-              pageLinks={pageLinks}
-              onCursor={this.onCursorChange}
-            />
-          </StreamContent>
-
-          <SidebarContainer showSidebar={isSidebarVisible}>
+                <PanelBody>
+                  <ProcessingIssueList
+                    organization={organization}
+                    projectIds={projectIds}
+                    showProject
+                  />
+                  {this.renderStreamBody()}
+                </PanelBody>
+              </Panel>
+              <StyledPagination
+                caption={tct('Showing [displayCount] issues', {
+                  displayCount,
+                })}
+                pageLinks={pageLinks}
+                onCursor={this.onCursorChange}
+              />
+            </Layout.Main>
+
             {/* Avoid rendering sidebar until first accessed */}
             {renderSidebar && (
```

Comment:
after closing the sidebar it just sits at the bottom of the page
![image](https://user-images.githubusercontent.com/1400464/145142436-cfb36c57-9843-434d-853f-93a20f57d899.png)

Reply:
lol. who put that there